src/mmw/js/src/modeling/views.js

```diff
@@ -66,10 +67,83 @@ var ModelingHeaderView = Marionette.LayoutView.extend({
 // The drop down containing the project name
 // and projects options drop down.
 var ProjectMenuView = Marionette.ItemView.extend({
-    model: models.ProjectModel,
+    ui: {
+        rename: '#rename-project',
+        share: '#share-project',
+        remove: '#delete-project',
+        print: '#print-project'
```

Comment:
I like IDs when we can use them because they're the fastest selector, I think attribute selectors are one of the slowest.  What's the case for a data attribute?